starknet_py/net/client_models.py

```diff
@@ -160,6 +160,15 @@ class TransactionReceipt:
     l1_to_l2_consumed_message: Optional[L1toL2Message] = None
 
 
+@dataclass
+class GatewayTransactionReceipt(TransactionReceipt):
+    """
+    Dataclass extending TransactionReceipt with code field
+    """
+
+    code: str = None
```

Comment:
FTR we were not specifically interested in the `code` value. So if you prefer to keep it simple, it's OK too :+1: